src/components/NetworkSelector/NetworkSelector.styled.ts

```diff
@@ -67,7 +67,7 @@ export const Option = styled.div`
     &.rinkeby {
       background: ${({ theme }): string => theme.yellow4};
     }
-    &.xdai {
+    &.gnosischain {
       background: ${({ theme }): string => theme.orange1};
```

Comment:
Maybe we need to change the Gnosis Chain color to #48a9a6
![image](https://user-images.githubusercontent.com/622217/146959648-391ca010-143d-4f29-a24c-e0e60a84dda0.png)

Reply:
And for the selector:
```
    background: rgb(72 169 166 / 25%);
    color: #48a9a
```
![image](https://user-images.githubusercontent.com/622217/146960223-e3761ad6-1ce7-41f2-852e-5365f8b491ef.png)

Reply:
I like it! It would make more sense with the brand. WDYT @alfetopito?

Reply:
Looks great!